plugins/utils_operators/directory_operator.py

```diff
@@ -37,17 +37,33 @@ class DeleteLocalDirectoryOperator(BaseOperator):
     """
 
     @apply_defaults
-    def __init__(self, path: str, delete_recursively: bool = True, **kwargs):
+    def __init__(self,
```

Comment:
I added the ability to this operator to take, as inputs, both task info and actual values